.github/PULL_REQUEST_TEMPLATE/new_feature.md

```diff
@@ -1,9 +1,17 @@
+---
+name: New Feature
+about: Add a new feature to the repository.
+title: 'New Feature: [NAME]'
+labels: 'enhancement'
+assignees: ''
+
+---
+
 ## This PR adds...
 
-*List your features here and their reasons for creation.*
+*List your features here and the benefits they bring.*
 
 ## Notes
 
-*List anything note-worthy here (potential issues, this needs merge to `master` before working, etc....).*
-
+*List anything note-worthy here (potential issues, this needs to merged to `master` before working, etc....).*
```

Comment:
```suggestion
*List anything note-worthy here (potential issues, this needs to be merged to `master` before working, etc....).*
```

Reply:
Oh yeah, you are right. It did look weird for me earlier but I couldn't put a finger on it